build.rs

```diff
@@ -41,6 +41,19 @@ mod musl_reference_tests {
         "rem_pio2.rs",
         "rem_pio2_large.rs",
         "rem_pio2f.rs",
+        "remquo.rs",
+        "remquof.rs",
+        "lgamma.rs",  // lgamma passed, lgamma_r has more than 1 result
```

Comment:
Could support for testing these functions that have more than one result be added? I think it'd be better than ignoring all of the files here

Reply:
Of course, improving test system is better. But I'm not sure I can do it.

Reply:
Stop. I forgot that these function have other signature in C:
`void sincos(double x, double *sin, double *cos)`
whereas in Rust we use
`fn sincos(x: f64) -> (f64, f64)`

Reply:
I am personally wary enough of these implementations that I do not want to land this PR without full tests for all added functions.

Reply:
I can add individual random tests for these functions to the files where they are implemented.